src/platform/i386/lapic.c

```diff
@@ -220,9 +241,35 @@ lapic_find_localaddr(void *l)
 	lapic = (void *)(addr);
 	printk("\tlapic: %p\n", lapic);
 
+	readmsr(MSR_APIC_BASE, &lo, &hi);
+	assert(lo & (1 << 8)); 	/* assume we are the BSP */
+	/* instead of using bit 11 ("enable"), we use the SIV LAPIC control register */
+
 	return addr;
 }
 
+void
+lapic_init(void)
+{
+	u32_t version;
+
+	assert(lapic);
+	lapic_write_reg(LAPIC_SIV_REG, LAPIC_SIV_ENABLE);
+
+	version = lapic_read_reg(LAPIC_VERSION_REG);
+	/* don't want to deal with booting up using CMOS */
```

Comment:
Is this a FIXME? or a TODO? Would be good to note that here.

Reply:
I will comment more details here.

Reply:
This comment was made by @gparmer, so I am not sure about the meaning of this comment here.
Maybe Gabe could explain more about this?